EIPS/eip-2537.md

```diff
@@ -383,6 +383,10 @@ Strictly following the spec will eliminate security implications or consensus im
 
 Important topic is a "constant time" property for performed operations. We explicitly state that this precompile **IS NOT REQUIRED** to perform all the operations using constant time algorithms.
 
+## Acknowledgements
```

Comment:
sorry this isn't a valid EIP section

Reply:
seen it. It seems odd is not allowed by EIPs though.  :)